### DISCUSSION TITLE
Dealing with Docker changes 🐳

### DISCUSSION THREAD

**Discussion by pransh15** [Unleash/unleash#3329] [General]
Hey Jonas, Starting a new discussion here, to keep discussions organised on GitHub and for our community to share their opinion on the topic.

From @jonasws on [this thread](https://github.com/orgs/Unleash/discussions/3260#discussioncomment-5322564):
> @pransh15 Congrats on the great numbers! Can you give any updates about how you're addressing the upcoming Docker Hub changes? https://blog.alexellis.io/docker-is-deleting-open-source-images/. Will you keep publishing to Docker Hub, or will you move to another hosting provider? Would be great with a heads up if we need to update our pipelines that pull your carefully crafted images :)

**Comment by rishiraj88** [comment]
Such a unwell move by the company behind Docker, if that's true.

**Comment by jonasws** [comment]
@pransh15 Seeing that Unleash seems to be quite well funded these days, would it make sense to just move to the paid plan?

It would also be really awesome if you started to "cross-publish" your images to the [AWS ECR Public Gallery](https://docs.aws.amazon.com/AmazonECR/latest/public/public-gallery.html), even though that may be a separate discussion.

**Comment by ivarconr** [comment]
yes, Unleash will go for a paid plan.

**Comment by testworksau** [comment]
Docker have backflipped:

https://www.docker.com/blog/no-longer-sunsetting-the-free-team-plan/
